Automattic/jetpack
# Contact form: remove double quotes in names in email headers

Fixes #7350

#### Changes proposed in this Pull Request:

In this PR, we remove the double quotes around names in email headers for emails sent via the Contact form module (e.g. `"Jeremy Herve" <wordpress@example.com>`, where the double quotes  should not be neccesary).
`Jeremy Herve <wordpress@example.com>` should be enough.

For more background around this, see the discussion in the original issue, #7350.

#### Jetpack product discussion

* Internal references: 3921793-zen, 6569-gh-jpop-issues

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

This may prove a bit tricky, as we'll need to test in different configurations to be sure this doesn't break things.

1. Go to Pages > Add New
1. Add a new contact form to your post content.
1. In the form settings, ensure the form is set to be sent to an email address you control.
1. Publish the page.
1. View the page, and submit a test form.
1. Ensure you receive the email from WordPress.
1. Copy the original full email contents.
1. Paste them [here](https://spamcheck.postmarkapp.com/), and ensure you get no errors.

One could also check when using a plugin like [this one](https://wordpress.org/plugins/mailin/) to send out the emails.
